app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -95,73 +103,15 @@ If you don’t want to come to England to study you can apply to train for inter
 
 For teachers who qualified outside the UK, as well as those with teaching experience, [other routes to qualified teacher status (QTS)](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk) are also open.
 
-## Need to know
-
-When you plan your training, bear in mind that:
-
-* international candidates can expect to pay an average of £14,765, plus living costs, to train 
-* depending on your immigration status and the subject you train to teach, you may get some financial support
-* training lasts one academic year full-time, or longer part-time
-* you can apply for paid teaching jobs as soon as you successfully complete your training
-* you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher
-* you can update, extend or switch your visa to live and work in the UK long-term
-
-
-
-## Apply for teacher training
-
-We can help you make an application for teacher training in England. Sign up for a teacher training adviser to get help right away and throughout the application process.
+## Check your qualifications and get help
 
```

Comment:
@laurahermionetennant92 it's not on here as you haven't made any amends to that bit but I saw it when I was looking at the page. In the section below this one in the Find out about fees and financial support section it says: 

If you’re planning on training to teach languages and physics. I think it should be languages **or** physics?

Reply:
Yes! Thank you